standards/collections.md

```diff
@@ -156,16 +156,17 @@ Reference the advantages of cursor-based pagination for some reasons why you may
 
 ### Guidelines
 
-- All collection-based endpoints **SHOULD** support offset-based over cursor-based pagination unless technology or performance makes cursor-based pagination more advantageous.
-- Offset-based collection endpoint responses **MUST** include the root element `paging` whenever `results` are used. 
+- While simple to implement, consider using cursor-based pagination instead
```

Comment:
I would follow the initial statement and adjust it to indicate that cursor-based pagination is required and that APIs may offer offset pagination if required.